src/table.js

```diff
@@ -194,6 +204,8 @@ export function Table(
 
   revalue();
 
+  setTimeout(updateSpacer, 25);
```

Comment:
This gives time for Observable to render the cell — I’m not sure if there’s a better or more supported way to defer this until the element is actually rendered. The `clientHeight` values are all zero until the table is placed into the DOM tree.